core/src/main/resources/xsl/util/funs.xsl

```diff
@@ -50,6 +50,12 @@
     <xsl:variable name="sink-types" as="xs:string*" select="('URL_FAIL', 'METHOD_FAIL', 'CONTENT_FAIL',
                                                              'REQ_TYPE_FAIL', 'ACCEPT')"/>
 
+    <!--
+        These are the types that result in an error.
+        For now it is all the sink types except for Accept.
+    -->
+    <xsl:variable name="error-sink-types" as="xs:string*" select="$sink-types[. != 'ACCEPT']"/>
```

Comment:
I would have done this the opposite way. I would have defined the `error-sink-types` and then made `sink-types` an additive composite instead of subtracting out all but `ACCEPT`. Either way, it works.